editor/event_listener_line_edit.cpp

```diff
@@ -91,6 +92,17 @@ String EventListenerLineEdit::get_event_text(const Ref<InputEvent> &p_event, boo
 		if (text.is_empty()) {
 			text = "(" + TTR("Unset") + ")";
 		}
+	} else if (mouse_motion.is_valid()) {
+		Vector2 rel = mouse_motion->get_relative();
+		if (rel.x < 0){
```

Comment:
```suggestion
		if (rel.x < 0) {
```
Please run `clang-format` as you have several errors like this and indentation errors

Reply:
My apologies. I'll get that setup and run later when I'm back on my personal machine.